data/_models/shared/Endpoint.csv

```diff
@@ -1,27 +1,27 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles
 Endpoint,,,,,,,,,,,schema:Service,General information describing an endpoint,"Beacon v2,FAIRDataPoint,DataCatalogueFlat"
-Endpoint,,id,string,1,TRUE,,,,,,"schema:identifier","A unique proper name or character sequence that identifies this particular person. In Beacon v2: id, in FAIR Genomes: PersonalIdentifier. In RD3: subjectID. In JRC-CDE: Pseudonym","Beacon v2,FAIRDataPoint,DataCatalogueFlat"
+Endpoint,,id,string,1,TRUE,,,,,,schema:identifier,"A unique proper name or character sequence that identifies this particular person. In Beacon v2: id, in FAIR Genomes: PersonalIdentifier. In RD3: subjectID. In JRC-CDE: Pseudonym","Beacon v2,FAIRDataPoint,DataCatalogueFlat"
 Endpoint,,type,hyperlink_array,,TRUE,,,,,,rdf:type,This row is an instance of one or more of these class(es) other than the Endpoint table.,"FAIRDataPoint,DataCatalogueFlat"
 Endpoint,,name,string_array,,TRUE,,,,,,dcterms:title,A name given to the resource.,"Beacon v2,FAIRDataPoint,DataCatalogueFlat"
-Endpoint,,version,text,,,,,,,,dcterms:hasVersion,An account of the resource version.,"DataCatalogueFlat"
+Endpoint,,version,text,,,,,,,,dcterms:hasVersion,An account of the resource version.,DataCatalogueFlat
 Endpoint,,description,text,,,,,,,,dcterms:description,An account of the resource.,"Beacon v2,FAIRDataPoint,DataCatalogueFlat"
 Endpoint,,publisher,ref_array,,TRUE,,Agent,,,,dcterms:publisher,An entity responsible for making the resource available.,"Beacon v2,FAIRDataPoint,DataCatalogueFlat"
```

Comment:
Move to organisation. Then this will point to a foaf:Organization (because that is the type of Organizations)

Reply:
don't forget to adapt demo data for this